plugins/modules/win_regmerge.ps1

```diff
@@ -28,15 +28,33 @@ $result = @{
 }
 $params = Parse-Args $args
 
-$path = Get-AnsibleParam -obj $params -name "path" -type "path" -failifempty $true -resultobj $result
+$path = Get-AnsibleParam -obj $params -name "path" -type "path" -resultobj $result
+$content = Get-AnsibleParam -obj $params -name "content" -type "content" -resultobj $result
 $compare_to = Get-AnsibleParam -obj $params -name "compare_to" -type "str" -resultobj $result
 
+if ( !$path -and !$content ) {
+    Fail-Json -obj $result -message "Missing required arguments: path or content. At lease one must be provided."
+}
+
+if (  $path -and $content ) {
+    Fail-Json -obj $result -message "Extra arguments: path or content. Only one must be provided."
+}
+
+if ( $content ) {
+    $guid = [guid]::NewGuid()
+    $path = $env:TEMP + "\" + $guid.ToString() + 'ansible_win_regmerge_content.reg'
+
+    $content | Out-File -FilePath $path
```

Comment:
Looks like `reg.exe IMPORT` doesn't care about the extension so we can simply do

```suggestion
    $path = [System.IO.Path]::GetTempFileName()
    Set-Content -LiteralPath $path -Value $content
```